src/android/project.rs

```diff
@@ -105,3 +127,21 @@ pub fn gen(
 
     Ok(())
 }
+
+fn write_asset_pack_build_file(dest: &Path, pack_name: &str) {
+    fs::write(
+        dest.join("build.gradle"),
+        &format!(
+            "apply plugin: 'com.android.asset-pack'
+
+assetPack {{
+    packName = \"{}\"
+    dynamicDelivery {{
+        deliveryType = \"install-time\"
```

Comment:
This should probably be in metadata together with the asset pack name. I have no way to currently test it, but better to be future-thinking on this.